packages/client/components/PhaseCompleted.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import PhaseCompleteTag from './Tag/PhaseCompleteTag'
+import UndoablePhaseCompleted from './UndoablePhaseCompleted'
+
+interface Props {
+  isComplete: boolean
+  canUndo?: boolean
+  meetingId?: string
+  resetToStageId?: string
+}
+
+const PhaseCompleted = (props: Props) => {
+  const {isComplete, canUndo, meetingId, resetToStageId} = props
+  if (!isComplete) return null
+  return (
+    <div>
+      {canUndo ? (
```

Comment:
`canUndo` is always true at the moment and `Wrapper` in `UndoablePhaseCompleted` seems to be identical to the `PhaseCompleteTag` styled component. Perhaps we could remove this file, use `UndoablePhaseCompleted` in the parent and use the `PhaseCompleteTag` in `UndoablePhaseCompleted` to reduce duplication?

Reply:
@nickoferrall that is true! i just pushed a change to pass in `canUndo=true` only from `retroGroupPhase`. Once Terry has the design, `UndoablePhaseCompleted` should have different styling to include a clickable button to undo (instead of using the hotkey).  I thinkk it's good to hold off on further refactoring until the UI design is settled on.

Reply:
Makes sense! Sounds good to me